site/content/docs/browser-checks/_index.md

```diff
@@ -38,9 +38,9 @@ We have stopped support for Puppeeteer with runtime 2022.10. [Read more about ou
 The following code is a valid Browser check using Playwright Test.
 
 {{< tabs "Basic example" >}}
-{{< tab "JavaScript" >}}
- ```js
-const { test } = require('@playwright/test')
+{{< tab "TypeScript" >}}
```

Comment:
Swapped the order of the tabs for TS being the default.

Reply:
👍